app/bibleview-js/src/components/TextEditor.vue

```diff
@@ -103,18 +113,50 @@ export default {
       title: 'Insert bible reference',
       result: async () => {
         const originalRange = document.getSelection().getRangeAt(0);
+        //Always add the "en" language, so at least the english parser always exists.
+        var en_parser = new bcv_parser;
+        var parsers = [en_parser]
```

Comment:
```suggestion
        const en_parser = new bcv_parser;
        const parsers = [en_parser]
```
My ide (webstorm) suggests these changes automatically. Which IDE are you using?

Reply:
I'm not using an IDE, just a text editor. This is a drawback but it has other advantages.